xml/art_sle_amazon_ecs.xml

```diff
@@ -132,6 +138,69 @@
           cluster. Remember to select the ECS role when launching the
           instance.
         </para>
+        <para>
+          You may use the Amazon EC2 Web Console to launch instances.
+          The SUSE Linux Enterprise Server ECS images are found in the
+          so called General Catalog. From the
+          <emphasis role="strong">Instances</emphasis> view select
+          <emphasis role="strong">Launche Instance</emphasis>. In the left
+          column select the <emphasis role="strong">Community AMIs</emphasis>
+          tab. Check the <emphasis role="strong">SUSE Linux</emphasis> box
+          on the left and enter <emphasis role="strong">ecs</emphasis> into
+          the search box. When configuring the instance parameters in the
+          <emphasis role="strong">Configure Instance Details</emphasis> view
+          select the <emphasis role="strong">ecsInstanceRole</emphasis> as
+          the IAM role. Then launch the instance.
+        </para>
+        <para>
+          Alternatively you can use the aws command line tool to
+          <link xlink:href="http://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html">run</link>
+          instances.
+        </para>
+        <screen>$ aws ec2 run-instances --image-id IMAGE_ID_SEE_PINT_OUTPUT --region EC2_REGION_NAME --key-name SSH_KEY_PAIR_NAME --instance-type DESIRED_INSTANCE_TYPE --subnet-id SUBNET_ID --iam-instance-profile 'Arn=PROFILE_PATH'</screen>
+        <para>
+          The PROFILE_PATH may be obtained by querying the IAM service.
+        <para>
+        <screen>$ aws iam list-instance-profiles</screen>
+        <para>
+          The SUBNET_ID will depend on your network configuration. If you
+          created your setup using the First Run Wizard you will most likely
+          want the instance to run in the subnet created by the wizard. Thus,
+          you can obtain the value by inspecting the properties of the running
+          ECS Instance.
+        </para>
+        <para>
+          Once the instance is running and if your cluster is named
+          <emphasis role="strong">default</emphasis> you can confirm that
+          the instance has attached itself to the cluster as follows:
+        </para>
+        <screen>$ aws ecs describe-clusters</screen>
+        <para>
+          The <emphasis role="strong">registeredContainerInstancesCount</emphasis>
```

Comment:
The "registeredContainerInstancesCount" is an entry in the json output produced by the command given in the <screen> section just prior to this. I think emphasis as in "look for this" are appropriate.